apps/web-giddh/src/app/invoice/recurring/recurring.component.html

```diff
@@ -122,9 +119,7 @@ <h4 class="modal-title bg pull-left">Search here...</h4>
                 <div class="form-group mrR1">
                     <label class="d-block mr-15">Last Invoice Date</label>
                     <div class="input-group">
-                        <input name="lastInvoiceDate" placeholder="Select Date" [(ngModel)]="filter.lastInvoiceDate" type="text"
-                            [bsConfig]="{dateInputFormat:'DD-MM-YYYY'}" autocomplete="off" class="form-control" bsDatepicker
-                            required>
+                        <giddh-datepicker name="lastInvoiceDate" [(ngModel)]="filter.lastInvoiceDate" placeholder="Select Date"></giddh-datepicker>    
```

Comment:
@shalinee-walkover  Please add square braces around placeholder and add single quotes around Select Date